packages/ag-charts-community/src/chart/interaction/interactionManager.ts

```diff
@@ -104,7 +98,11 @@ export class InteractionManager extends BaseManager<
     private touchDown = false;
     private dragStartElement?: HTMLElement;
 
-    private pausers: Record<PauseType, number> = { animation: 0, 'context-menu': 0 };
+    private stateQueue: Set<InteractionState> = new Set();
+    private _state: InteractionState = InteractionState.Default;
+    public get state() {
```

Comment:
This is probably not needed if we make filtering by state for listeners a responsibility of `InteractionManager`?

Reply:
This is related to https://github.com/ag-grid/ag-charts/pull/906/files#r1472880057, so I'll resolve this convo.